src/modules/accounting/accounting.py

```diff
@@ -125,6 +134,11 @@ def _calculate_report(self, blockstamp: ReferenceBlockStamp) -> ReportData:
             extra_data_items_count=extra_data.items_count,
         )
 
+        ACCOUNTING_IS_BUNKER.set(report_data.is_bunker)
```

Comment:
Maybe have separate function where all metrics from report_data will be set?
like
```python
def set_mertrics(data: ReportData):
```